src/org/usfirst/frc/team467/robot/Autonomous/Actions.java

```diff
@@ -194,6 +165,14 @@ public static ActionGroup start() {
 		return mode;
 	}
 	
+	public static ActionGroup goStraight(){
```

Comment:
How about a more explicit name, like "crossAutoLine"? The name "goStraight" doesn't really imply a strategy, rather just a step in some other more complex strategy.